src/Microsoft.PowerShell.SDK/Microsoft.PowerShell.SDK.csproj

```diff
@@ -15,20 +15,21 @@
   </ItemGroup>
 
   <ItemGroup>
+    <PackageReference Include="Microsoft.Windows.Compatibility" Version="2.0.0" />
```

Comment:
If we don't know the source location of this package, maybe put it at the end, in the section with the comment ` the source could not be found for the following package(s)`, along with `Microsoft.NETCore.Windows.ApiSets`

Reply:
Ok